liquidation-monitor/src/agent.js

```diff
@@ -0,0 +1,500 @@
+    // topic0: 0x13ed6866d4e1ee6da46f845c46d7e54120883d75c5ea9a2dacc1c4ca8984ab80
+
+    // Simple bootstrap with Compound for now
+    const apiURL = 'https://api.compound.finance/api/v2/account';
+
+    // Helper for Compound API
+    function buildJsonRequest(maxHealth, minBorrow, pageNumber, pageSize) {
+      const jsonRequest = {
+        addresses: [], // returns all accounts if empty or not included
+        block_number: 0, // returns latest if given 0
+        max_health: { value: maxHealth },
+        min_borrow_value_in_eth: { value: minBorrow },
+        page_number: pageNumber,
+        page_size: pageSize,
+      };
+      return jsonRequest;
+    }
+
+    // Find total number of results with the first request
+    const initialRequest = buildJsonRequest(
+      maximumHealth,
+      minimumBorrowInETH,
+      1,
+      1,
+    );
+    const initialResults = await callAPI(apiURL, initialRequest);
+    const totalEntries = initialResults.pagination_summary.total_entries;
+
+    // Determine number of pages needed to query. Results vs config limit.
+    const maxEntries = Math.min(maxTrackedAccounts, totalEntries);
+    const maxPages = Math.ceil(maxEntries / 100);
+
+    // Query each page and add accounts. Starting at 1 and including maxPages
+    const accounts = [];
+    const pages = [...Array(maxPages)].map((_, i) => 1 + i);
+    await Promise.all(pages.map(async (page) => {
+      const currentRequest = buildJsonRequest(
+        maximumHealth,
+        minimumBorrowInETH,
+        page,
+        100, // 100 results per page was optimal
+      );
+      const apiResults = await callAPI(apiURL, currentRequest);
+      apiResults.accounts.forEach((account) => {
+        accounts.push(account);
+      });
+    }));
+    // #endregion
+
+    // #region Parse Compound data into new objects
+    // Get a unique list of token addresses
+    ts('Processing tokens now');
+    const tokenSet = new Set();
+    accounts.forEach((account) => {
+      account.tokens.forEach(async (token) => {
+        tokenSet.add(token.address);
+      });
+    });
+
+    // Initialize token objects first
+    await Promise.all(Array.from(tokenSet).map(async (token) => {
+      await verifyToken(data, token);
+    }));
+
+    // Loop through found accounts
+    ts('Updating user balances');
+    accounts.forEach((account) => {
+      // add to tracked accounts
+      /* eslint-disable no-param-reassign */
+      if (data.accounts[account.address] === undefined) {
+        data.accounts[account.address] = {};
+      }
+      // Add found health
+      data.accounts[account.address].health = BigNumber(account.health.value);
+      // Loop through tokens and update balances.
+      account.tokens.forEach((token) => {
+        // Process borrows as 'token'
+        /* eslint-disable eqeqeq */
+        if (
+          token.borrow_balance_underlying !== undefined
+          && token.borrow_balance_underlying.value != 0
```

Comment:
Is this intended to be `!=` or `!==`?
```suggestion
          && token.borrow_balance_underlying.value !== 0
```